corehq/ex-submodules/casexml/apps/case/models.py

```diff
@@ -229,6 +229,10 @@ def closed_transactions(self):
         return filter(lambda action: action.action_type == const.CASE_ACTION_CLOSE, self.actions)
 
     @property
+    def transactions(self):
```

Comment:
right i think i caused a bug from this a while back, but wasn't quite sure that that was the reason. makes sense. will update to modify the SQL model instead